docs/Listener.md

```diff
@@ -0,0 +1,29 @@
+
+# Kyma Listener
+
+Listener component that listens to events sent by the Kyma [watcher](https://github.com/kyma-project/kyma-watcher) component.
```

Comment:
are you using the words "component" and "module" as synonyms here, or is "listener component" (line 4) something different from "listener module" (line 8)? 

```suggestion
Listener component that listens to events sent by the Kyma [Watcher](https://github.com/kyma-project/kyma-watcher) component.
```
In the [Watcher README](https://github.com/kyma-project/kyma-watcher/tree/e84b0a21696f5b13e50451990bfb332cfe0f9ea7#readme), you capitalize the "Watcher". We should stay consistent here.
In a related note, should the "listener" be capitalized as well, or is there a reason to keep it lower case? Or shall we use lower case for both, "watcher" and "listener"?

Reply:
Thank you for pointing this out! Both should be capitalized, and the listener is a module, not a separate component.
I have changed all **_listener component_** occurrences to **_Listener module_**.